### ISSUE TITLE
How to see the result?

### ISSUE THREAD

**Issue by goalseep** [endernewton/tf-faster-rcnn#427]
When I run the demo.py according to README.md, it show:

2019-03-11 11:45:41.956370: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2300000000 Hz
2019-03-11 11:45:41.956754: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x1b2a7e0 executing computations on platform Host. Devices:
2019-03-11 11:45:41.956807: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
2019-03-11 11:45:42.114236: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:998] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2019-03-11 11:45:42.114929: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x1b2a520 executing computations on platform CUDA. Devices:
2019-03-11 11:45:42.114994: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): Tesla K80, Compute Capability 3.7
2019-03-11 11:45:42.115455: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1433] Found device 0 with properties: 
name: Tesla K80 major: 3 minor: 7 memoryClockRate(GHz): 0.8235
pciBusID: 0000:00:04.0
totalMemory: 11.17GiB freeMemory: 11.10GiB
2019-03-11 11:45:42.115493: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1512] Adding visible gpu devices: 0
2019-03-11 11:45:43.500333: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-03-11 11:45:43.500405: I tensorflow/core/common_runtime/gpu/gpu_device.cc:990]      0 
2019-03-11 11:45:43.500429: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1003] 0:   N 
2019-03-11 11:45:43.500792: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10754 MB memory) -> physical GPU (device: 0, name: Tesla K80, pci bus id: 0000:00:04.0, compute capability: 3.7)
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /content/drive/Colab/tf-faster-rcnn/tools/../lib/nets/network.py:213: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
WARNING:tensorflow:From /content/drive/Colab/tf-faster-rcnn/tools/../lib/layer_utils/proposal_layer.py:75: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
WARNING:tensorflow:From /usr/local/lib/python3.6/dist-packages/tensorflow/python/training/saver.py:1266: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
Loaded network output/res101/voc_2007_trainval+voc_2012_trainval/default/res101_faster_rcnn_iter_110000.ckpt
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Demo for data/demo/000456.jpg
2019-03-11 11:45:59.134886: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_12. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.134969: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_14. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.135012: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_16. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.135058: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_18. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.426327: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_12. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.426435: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_14. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.426478: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_16. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.426545: W ./tensorflow/core/grappler/optimizers/graph_optimizer_stage.h:241] Failed to run optimizer ArithmeticOptimizer, stage RemoveStackStridedSliceSameAxis node resnet_v1_101_3/rois/strided_slice_18. Error: Pack node (resnet_v1_101_3/rois/stack) axis attribute is out of bounds: 1
2019-03-11 11:45:59.631511: I tensorflow/stream_executor/dso_loader.cc:152] successfully opened CUDA library libcublas.so.10.0 locally
Detection took 7.069s for 300 object proposals
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Demo for data/demo/000542.jpg
Detection took 0.380s for 300 object proposals
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Demo for data/demo/001150.jpg
Detection took 0.374s for 300 object proposals
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Demo for data/demo/001763.jpg
Detection took 0.383s for 300 object proposals
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Demo for data/demo/004545.jpg
Detection took 0.371s for 300 object proposals
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
<Figure size 1200x1200 with 1 Axes>
